### DISCUSSION TITLE
0.0.15b1

### DISCUSSION THREAD

**Discussion by Hellowlol** [custom-components/nordpool#402] [Announcements]
## What's Changed
* Fixes non async call async_forward_entry_setup by @rrooggiieerr in https://github.com/custom-components/nordpool/pull/390
* Remove unnecessary call to template.attach by @emontnemery in https://github.com/custom-components/nordpool/pull/397
* Update Finnish VAT by @vla89 in https://github.com/custom-components/nordpool/pull/394
* Update aio_price.py by @Hellowlol in https://github.com/custom-components/nordpool/pull/400
* Update __init__.py by @Hellowlol in https://github.com/custom-components/nordpool/pull/401

## New Contributors
* @emontnemery made their first contribution in https://github.com/custom-components/nordpool/pull/397
* @vla89 made their first contribution in https://github.com/custom-components/nordpool/pull/394

**Full Changelog**: https://github.com/custom-components/nordpool/compare/0.0.15b0...0.0.15b1

<hr /><em>This discussion was created from the release <a href='https://github.com/custom-components/nordpool/releases/tag/0.0.15b1'>0.0.15b1</a>.</em>

**Comment by jkaljundi** [comment]
After installing all my HA nordpool entities are unavailable, even after restart;

nordpool_kwh_ee_eur_4_055_02
This entity is no longer being provided by the nordpool integration. If the entity is no longer in use, delete it in settings.

After some digging found they have been renamed. The new entity name has an additional 2 at the end of it: 
nordpool_kwh_ee_eur_4_055_022

**Comment by Kattihatt** [comment]
Having the same issue. Doesn't work after update.
